x/auth/client/testutil/helpers.go

```diff
@@ -62,12 +62,8 @@ func TxMultiSignExec(clientCtx client.Context, from, filename string, extraArgs
 	return clitestutil.ExecTestCLICmd(clientCtx, cli.GetMultiSignCommand(), append(args, extraArgs...))
 }
 
-func TxSignBatchExec(clientCtx client.Context, from fmt.Stringer, filename string, extraArgs ...string) (testutil.BufferWriter, error) {
-	args := []string{
-		fmt.Sprintf("--from=%s", from.String()),
-		filename,
-	}
-
+func TxSignBatchExec(clientCtx client.Context, from fmt.Stringer, filenames []string, extraArgs ...string) (testutil.BufferWriter, error) {
```

Comment:
We cannot introduce a breaking change in patch release, so let's revert this here and on main (as the breaking change wasn't necessary at the first place).
For the test case, we can just use `clitestutil.ExecTestCLICmd` directly and leave the API unchanged for the others.